lib/modules/manager/npm/index.ts

```diff
@@ -20,7 +20,11 @@ export const url = 'https://docs.npmjs.com';
 export const categories: Category[] = ['js'];
 
 export const defaultConfig = {
-  fileMatch: ['(^|/)package\\.json$', '(^|/)pnpm-workspace\\.yaml$'],
+  fileMatch: [
+    '(^|/)package\\.json$',
+    '(^|/)package\\.yaml$',
```

Comment:
I would prefer to revert this change and keep it as-is initially. You and any others can modify `fileMatch` for now and if we find no unintended consequences then it can be added as a default file match.

If we keep the fileMatch unchanged, we can merge this if we're confident it doesn't cause regressions for JSON. But otherwise we need to be more careful / slower to approve.

Reply:
One question, should I add somewhere in the logs that this is possible? Otherwise any other person using the yaml and trying to use renovate will think yaml is not supported.

Reply:
Add in the `npm/readme.md` file